### PR TITLE
Migrate testing projects to JVM, software platforms

### PR DESCRIPTION
**Pull request by tresat**

### CODE REVIEW

**Comment by tresat** [comment]
@bot-gradle test this

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -67,9 +67,6 @@ subprojects/jvm-services/                   @gradle/bt-jvm
 subprojects/normalization-java/             @gradle/bt-jvm
 subprojects/platform-jvm/                   @gradle/bt-jvm
 subprojects/plugins/                        @gradle/bt-jvm
-subprojects/testing-junit-platform/         @gradle/bt-jvm @gradle/ge-testing
-subprojects/testing-jvm/                    @gradle/bt-jvm @gradle/ge-testing
-subprojects/testing-jvm-infrastructure/     @gradle/bt-jvm @gradle/ge-testing
```
Comment (big-guy) [inline .github/CODEOWNERS:72]: We need to preserve these owners for GE

**Review by big-guy — CHANGES_REQUESTED** [review]
One comment about owners. Moves LGTM.

**Comment by tresat** [comment]
@bot-gradle test and merge

**Pull request merged**